packages/open-truss/src/components/index.ts

```diff
@@ -1 +1,3 @@
-export { default as OTExampleComponent } from './OTExampleComponent'
+import * as _OTExampleComponent from './OTExampleComponent'
+export const OTExampleComponent = _OTExampleComponent
```

Comment:
This is a breaking change, as anyone who was importing this component directly would need to do `OTExampleComponent.default` now, but I'm not worried about it.